assets/src/styles/blocks/ShareButtons/ShareButtonsStyles.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../master-theme/assets/src/scss/base/colors";
+
+@mixin sharedStyles($color) {
+  background: $color;
+
+  &:hover {
+    background: darken($color, 20%);
+  }
+
+  &:active {
```

Comment:
Can you remove this from the PR to keep things simple? We don't have a spec for the active state yet.